lib/plugin/allure.js

```diff
@@ -74,22 +74,40 @@ const defaultConfig = {
 module.exports = (config) => {
   config = Object.assign(defaultConfig, config);
 
+  const plugin = {};
+
   const reporter = new Allure();
   reporter.setOptions({ targetDir: config.outputDir });
 
   let currentMetaStep = [];
   let currentStep;
   let isHookSteps = false;
 
-  this.addAttachment = (name, buffer, type) => {
+  reporter.pendingCase = function (testName, timestamp, opts = {}) {
+    reporter.startCase(testName, timestamp);
+
+    if (opts.description) plugin.setDescription(opts.description);
+    if (opts.severity) plugin.severity(opts.severity);
+    if (opts.severity) plugin.addLabel('tag', opts.severity);
+
+    reporter.endCase('pending', { message: opts.message || 'Test ignored' }, timestamp);
```

Comment:
what do you think if we use `Pending Test` to make it more consistent :)

Reply:
'Test ignored' - is default message from allure-js-common)

Reply:
Didn't know that 😺